RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+This release updates the documentation.
+An example of error response expansion was added to the guide.
```

Comment:
I think we could skip the release for this documentation change, unless you like my idea below :D